docker/entrypoint.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/sh
 
-## Allow us to split the $SEBAK_VALIDATORS 'array' into multiple `--validator` args
-## Note: It's not a proper array since the shell we use is `sh`, not `bash`
-./sebak genesis ${SEBAK_GENESIS_BLOCK}
-for VALIDATOR in ${SEBAK_VALIDATORS}; do
-    VALIDATOR_ARGS="${VALIDATOR_ARGS} --validator=${VALIDATOR}"
-done
-./sebak node ${VALIDATOR_ARGS} --network-id=${SEBAK_NETWORK_ID} --secret-seed=${SEBAK_SECRET_SEED} --log-level debug
+## This entrypoint has 2 modes: if any argument is provided to `docker run`,
+## the arguments are passed directly to sebak
+## Otherwise, it just starts a node with the environment file
+if [ $# -gt 1 ]; then
+    ## Argument mode
+    ./sebak $@
```

Comment:
Good feature. If extra options is given, the `SEBAK_XX` env variables seem to be ignored. It will be pressure to set the entire options to add one option.

Reply:
I just tested it, the environment variable are not ignored. `VALIDATOR_ARGS` is ignored, since it's not a native env variable for sebak, but `SEBAK_NETWORK_ID` for example will work.

Note that the first mode is designed to quick CLI testing, while the second mode is more useful if you want to spawn a network. For example @cmcm2222 and I use this mode to test our implementation of #1 quickly.

Reply:
@Geod24 yes, I was wrong, `sebak` already knows the env variables :)